spicelib/editor/spice_editor.py

```diff
@@ -318,12 +325,16 @@ def get_subcircuit(self, instance_name: str) -> 'SpiceCircuit':
                 if m:  # If it is a library include
                     lib = m.group(2)
                     if os.path.exists(lib):
-                        lib_filename = os.path.join(os.path.expanduser('~'), "Documents/LTspiceXVII/lib/sub", lib)
-                        if os.path.exists(lib_filename):
-                            sub_circuit = SpiceEditor.find_subckt_in_lib(lib_filename, subcircuit_name)
-                            if sub_circuit:
-                                break
-                    for path in LibSearchPaths:
+                        # TODO: this is geared towards LTspice ("/sub"). We should have a more general way to find the sub-circuit
```

Comment:
I agree, I think that the join with the sub is not needed. The inconvenient is that it will be slower, but will cover more situations.
The .lib could be anywhere.

Reply:
working on it, with ngspice. will look a bit more.

Reply:
made like the others